test/Microsoft.AspNet.Razor.Test/Framework/BlockTypes.cs

```diff
@@ -177,27 +177,43 @@ public MarkupBlock(IEnumerable<SyntaxTreeNode> children)
 
     public class MarkupTagHelperBlock : TagHelperBlock
     {
-        public MarkupTagHelperBlock(string tagName)
-            : this(tagName, new Dictionary<string, SyntaxTreeNode>())
+        public MarkupTagHelperBlock(string tagName, bool selfClosing = false)
```

Comment:
no need for default parameter; just add another constructor overload that doesn't have a `selfClosing` parameter